ui/src/layouts/Baseline.vue

```diff
@@ -269,8 +277,16 @@ export default {
     },
     mounted () {
         this.updateTheme()
+        this.checkMobile()
+        window.addEventListener('resize', this.checkMobile)
+    },
+    beforeUnmount () {
+        window.removeEventListener('resize', this.checkMobile)
     },
     methods: {
+        checkMobile () {
+            this.isMobile = window.innerWidth < 768
```

Comment:
Magic number is somewhat arbitrary and not conducive to being "mobile"

What is the logic here? Is it just that the screen is sufficiently small (regardless of being mobile) to trigger this logic?

Reply:
> What is the logic here? Is it just that the screen is sufficiently small (regardless of being mobile) to trigger this logic?

Yes, "is mobile" isn't really the right term, more "is small", but its a breakpoint that's synonymous with mobile.